src/commands/god.ts

```diff
@@ -97,10 +98,18 @@ const godCommand = new Command({
       return;
     }
     logger.debug("God info found ", god);
+
+    var abilities = god.abilities;
```

Comment:
Don't use `var`, you want to use `const` unless you need to change the reference, then use `let`